Library/Homebrew/keg.rb

```diff
@@ -339,14 +339,14 @@ def to_formula
   end
 
   def installed_dependents
+    return [] if !linked? && !to_formula.keg_only?
```

Comment:
Let's go for `unless optlinked?` rather than `if !linked? && !to_formula.keg_only?`